frontend/src/js/store/onboardingSlice/thunks.ts

```diff
@@ -70,7 +72,7 @@ const deductOnboardingState = ({ devicesById, devicesByStatus, onboardingState,
     ),
     showTips: onboardingState.showTips != null ? onboardingState.showTips : true,
     deviceType,
-    approach: onboardingState.approach || (deviceType.some(type => type.startsWith('qemu')) ? 'virtual' : 'physical'),
```

Comment:
Can't we rely on the device type `generic-x86_64`? If the issue is that the name is not appropriate (too generic :sweat_smile: ) we can change it to something explicit like `virtual-device-x86_64` or whatever you suggest.

So far this has been our internal image, and now that we are about to expose it to the users we have the opportunity to improve UX.

Reply:
my assumption was with `generic-x86_64` there would be a high chance there's other users as well that land on this as a fallback - if that's not the case I'll happily use that!